SharedPackages/DataBrokerProtectionCore/Tests/DataBrokerProtectionCoreTests/BrokerProfileJobQueueModeTests.swift

```diff
@@ -88,26 +88,15 @@ final class BrokerProfileJobQueueModeTests: XCTestCase {
         XCTAssertFalse(result)
     }
 
-    func testWhenModeIsIdle_thenPriorityDateIsNil() throws {
```

Comment:
I don't know why there was a test for this, the priority date in idle mode is completely academic